install_files/ansible-base/roles/prepare-servers/tasks/main.yml

```diff
@@ -49,3 +49,11 @@
     purge: yes
   tags:
     - apt
+
+- name: Upgrade all packages
+  apt:
+    name: "*"
+    state: latest
+    update_cache: yes
```

Comment:
The more obvious way to do this would be 

```
apt:
  upgrade: dist
  update_cache: yes
```

which is what we do here: https://github.com/freedomofpress/securedrop/blob/c267329e5bad9c3396a5087b41275bb324fbf4f8/install_files/ansible-base/roles/common/tasks/apt_upgrade.yml#L45-L50, essentially. Mind updating to that syntax just to make it clear there's no apt magic happening with the wildcard?

Reply:
I'm explicitly not doing a dist-upgrade because I don't want unexpected state changes in terms of which packages are installed/removed. But that may just be paranoia, I don't think there's a convincing rationale for it. Let's see how it behaves as a `dist`.